docs/components/tree-list.md

```diff
@@ -215,6 +215,26 @@ Boolean to hide icons. Defaults to `false`.
 <KTreeList hide-icons :items="items" />
 ```
 
+### collapsable
+
+Boolean to enable/disable collapse feature. Defaults to `false`.
+
+<KTreeList :collapsable="true" :items="collapsableItems" />
+
+```html
+<KTreeList :collapsable="true" :items="items" />
+```
+
+### collapseAll
+
+Boolean to collapse/expand all TreeListItems. Should be used only with `collapsable` prop. Defaults to `false`.
```

Comment:
question: does this mean just "initially collapse" all?

Reply:
I think that’s a bit odd because it’s neither a default value nor a controlled prop.

If it’s meant to be a default value, it should be overridden by internal state changes (e.g., when users expand or collapse nodes).
If it’s a controlled prop, then `:collapse-all="true"` should ensure all nodes are collapsed.

Reply:
@DariaYeremina can we add two new methods in the component, `expandAll()` (expands all collapsable items) and `collapseAll()` (collapses all collapsable items), add them to `defineExpose` and the documentation?

Then, the prop name `collapseAll` could be renamed to `collapse-all-initial` (or something) and remove the watcher?